plugins/source/aws/resources/services/ram/resource_types.go

```diff
@@ -12,37 +12,21 @@ func ResourceTypes() *schema.Table {
 		Name:        "aws_ram_resource_types",
 		Description: `https://docs.aws.amazon.com/ram/latest/APIReference/API_ServiceNameAndResourceType.html`,
 		Resolver:    fetchRamResourceTypes,
-		Transform:   transformers.TransformWithStruct(&types.ServiceNameAndResourceType{}),
-		Multiplex:   client.ServiceAccountRegionMultiplexer("ram"),
+		Transform: transformers.TransformWithStruct(&types.ServiceNameAndResourceType{},
+			transformers.WithPrimaryKeys("ResourceType", "ServiceName")),
+		Multiplex: client.ServiceAccountRegionMultiplexer("ram"),
 		Columns: []schema.Column{
 			{
-				Name:     "account_id",
-				Type:     schema.TypeString,
-				Resolver: client.ResolveAWSAccount,
-				CreationOptions: schema.ColumnCreationOptions{
-					PrimaryKey: true,
-				},
+				Name:            "account_id",
+				Type:            schema.TypeString,
+				Resolver:        client.ResolveAWSAccount,
+				CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
 			},
 			{
-				Name:     "region",
-				Type:     schema.TypeString,
-				Resolver: client.ResolveAWSRegion,
-			},
-			{
-				Name:     "resource_type",
-				Type:     schema.TypeString,
-				Resolver: schema.PathResolver("ResourceType"),
-				CreationOptions: schema.ColumnCreationOptions{
-					PrimaryKey: true,
-				},
-			},
-			{
-				Name:     "service_name",
-				Type:     schema.TypeString,
-				Resolver: schema.PathResolver("ServiceName"),
-				CreationOptions: schema.ColumnCreationOptions{
-					PrimaryKey: true,
-				},
+				Name:            "region",
+				Type:            schema.TypeString,
+				Resolver:        client.ResolveAWSRegion,
+				CreationOptions: schema.ColumnCreationOptions{PrimaryKey: true},
```

Comment:
What if a service has a global + regional resources (WAF)?

Reply:
But both can be in the same region... So would create a duplicate

Reply:
I assume waf & waf regional will have different `service_name`